packages/hoppscotch-common/src/components/graphql/Authorization.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="flex flex-col flex-1">
     <div
-      class="sticky z-10 flex items-center justify-between flex-shrink-0 pl-4 overflow-x-auto border-b bg-primary border-dividerLight"
+      class="sticky z-10 flex items-center justify-between flex-shrink-0 pl-4 overflow-x-auto border-b bg-primary border-dividerLight top-[2.1rem]"
```

Comment:
@anwarulislam, don't use hardcoded values here. Use suitable CSS variables from themes.scss.

Reply:
I'll fix this part.